ros2_batch_job/__main__.py

```diff
@@ -66,7 +66,7 @@
     'pydocstyle',
     'pyflakes',
     'pyparsing',
-    'pytest',
+    'pytest==4.6.4',
```

Comment:
Maybe `pytest<5`? Might need double-quotes, depending on how pip is invoked. Just a suggestion.

Reply:
Thanks for the suggestion. I don't see any 3.x releases after the 4.0 nor 2.x releases after the 3.0 so I don't expect to see any 4.x release now that 5.0 is out. I'm also hoping that we'll have a fix upstream within a reasonable time.